Sources/App/Services/TerminalService.swift

```diff
@@ -0,0 +1,114 @@
+import BuildToolchainEngine
+import TSCBasic
+import Vapor
+
+public final class TerminalService {
```

Comment:
Vapor does not (to my knowledge) have concept of "web-socket controller", so this class follows Kitura's API.